modules/javafx.controls/src/test/java/test/javafx/scene/control/ToggleButtonTest.java

```diff
@@ -42,6 +42,7 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import de.sandec.jmemorybuddy.JMemoryBuddy;
```

Comment:
Throws compilation error:  package de.sandec.jmemorybuddy does not exist

Reply:
fixed